pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtl.java

```diff
@@ -267,18 +266,24 @@ private static void validateOptions(FhirEtlOptions options)
   // #278).
   static Pipeline buildHapiJdbcFetch(
       FhirEtlOptions options, DatabaseConfiguration dbConfig, FhirContext fhirContext)
-      throws PropertyVetoException {
+      throws PropertyVetoException, SQLException {
     boolean foundResource = false;
     Pipeline pipeline = Pipeline.create(options);
     JdbcConnectionUtil jdbcConnectionUtil = createJdbcConnection(options, dbConfig);
 
-    FhirSearchUtil fhirSearchUtil = createFhirSearchUtil(options, fhirContext);
+    Map<String, Integer> resourceCount = null;
+    if (!options.isRunIncremental()) {
```

Comment:
we are basing our assumption that if its not going to be incremental run then use fhir search and if it is then use jdbc fetch. should we add this as comment or should be use better variable name?

Reply:
renamed the variable and added a comment